ts/components/loader.ts

```diff
@@ -150,26 +150,26 @@ export const PathFilters: { [name: string]: PathFilterFunction } = {
 };
 
 /**
- * The implementation of the dynamic loader
+ * The version of MathJax that is running.
  */
-export namespace Loader {
-  /**
-   * The version of MathJax that is running.
-   */
-  const VERSION = MJGlobal.version;
+const VERSION = MJGlobal.version;
 
-  /**
-   * The versions of all the loaded extensions.
-   */
-  export const versions: Map<string, string> = new Map();
+/**
+ * The versions of all the loaded extensions.
+ */
+const versions = new Map<string, string>();
```

Comment:
I would like this to be exported as well, as I plan to use this in the future to make a better list of components in the "About MathJax" menu item.  Or it could be in the `Loader` constant below, in which case, you need `this.versions` in the `load()` and `saveVersion()` methods below.

Reply:
Included `versions` in `Loader`, but referenced as `Loader.versions` as v8 can not optimize objects with `this` references. (I learned that at Google IO 2013, so this might be outdated knowledge.)